src/app/game/model/data/ScenarioRule.ts

```diff
@@ -41,7 +41,7 @@ export class MonsterSpawnData {
 export class ScenarioFigureRule {
 
   identifier: FigureIdentifier = undefined;
-  type: "present" | "dead" | "gainCondition" | "looseCondition" | "damage" | "hp" | "toggleOff" | "toggleOn" | "transfer" | "remove" | "amAdd" | "amRemove" = "present";
+  type: "present" | "dead" | "gainCondition" | "looseCondition" | "damage" | "hp" | "discard" | "toggleOff" | "toggleOn" | "transfer" | "remove" | "amAdd" | "amRemove" = "present";
```

Comment:
Is this meant as scenario effect to discard card before? Then labels must be added for this rule to show properly! Otherwise rule will just show: 
> discard as scenario effect